### PR TITLE
Update categories page to include budget in chart

### PR DESCRIPTION
**Pull request by vtopham**
https://trello.com/c/iY4Vyibr

This adds a reducer for the budget and allows for it to be shown comparing the budget to expenditure by category. This is an MVP solution, if we want something prettier we can work on some ideas :)

### CODE REVIEW

File: src/components/pages/CategorizedSpending/CategorizedSpending.js [hunk 1]
```diff
@@ -137,7 +138,12 @@ const CategorizedSpending = props => {
           <StyledDiv>
             <h1>Spending By Category</h1>
             <h1>Your Spending For {chosenMonth}</h1>
-            <CategorizedPlot
+            {/* <CategorizedPlot
```
Comment (Cireimu) [inline src/components/pages/CategorizedSpending/CategorizedSpending.js]: Is there any reason this code is commented out? If not then please remove it.

**Review by Cireimu — APPROVED** [review]
Looks good to go, just a few commented out pieces of code I'm unsure about but beyond that, it looks good.

File: src/components/pages/CategorizedSpending/CategorizedSpending.js (hunk 1 above)
Comment (vtopham) [reply src/components/pages/CategorizedSpending/CategorizedSpending.js]: This code has been deleted

**Review by locbgiang — APPROVED** [review]
Looks good.

**Pull request merged**